x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.test.ts

```diff
@@ -436,12 +443,19 @@ describe('Log threshold executor', () => {
             },
           },
         ]);
+
+        expect(alertLimitMock.setLimitReached).toHaveBeenCalledWith(false);
```

Comment:
The naming of the spec doesn't make it clear to me why ungrouped would be `setLimitReached: false` and grouped would be `setLimitReached: true`. But I may be lacking some common context.

Reply:
Good point, I'll look into splitting up the test cases.